src/lib/power-to-energy/index.ts

```diff
@@ -0,0 +1,54 @@
+import {z} from 'zod';
+
+import {ERRORS} from '@grnsft/if-core/utils';
+import {ConfigParams, PluginParams} from '@grnsft/if-core/types';
+import {PluginFactory} from '@grnsft/if-core/interfaces';
+
+import {validate} from '../../../common/util/validations';
+
+import {STRINGS} from '../../config';
```

Comment:
This module does not exist in [if-plugin-template](https://github.com/Green-Software-Foundation/if-plugin-template). You should fix with other solution.